app/uk/gov/hmrc/plasticpackagingtax/registration/config/AppConfig.scala

```diff
@@ -126,4 +126,9 @@ class AppConfig @Inject() (config: Configuration, val servicesConfig: ServicesCo
   def unauthenticatedFeedbackUrl(): String =
     s"$feedbackUnauthenticatedLink?service=${serviceIdentifier}"
 
+  lazy val isLiabilityPreLaunchEnabled: Boolean =
+    config
+      .getOptional[Boolean](s"features.liabilityPreLaunch")
+      .getOrElse(true)
```

Comment:
I think we should default to `false`, ie. default behaviour is that of post launch.

Reply:
Addressed.